src/components/StatsTooltip/ChainsStatsTooltipRow.tsx

```diff
@@ -5,48 +5,45 @@ import "./StatsTooltip.css";
 import { formatAmount } from "lib/numbers";
 
 type Props = {
-  title: string;
-  total: BigNumber;
-  avaxValue: BigNumber;
-  arbitrumValue: BigNumber;
+  entries: { [key: string]: BigNumber | string | undefined };
   showDollar?: boolean;
   decimalsForConversion?: number;
   symbol?: string;
   shouldFormat?: boolean;
 };
 
 export default function ChainsStatsTooltipRow({
-  title,
-  total,
-  avaxValue,
-  arbitrumValue,
+  entries,
   showDollar = true,
   decimalsForConversion = USD_DECIMALS,
   symbol,
   shouldFormat = true,
 }: Props) {
+  const validEntries = Object.entries(entries).filter(
```

Comment:
why there entries is used if only values needed?
maybe change to Object.values?

Reply:
Using entries I am creating an array of all the valid entries based on the value being truthy. So, if the value is falsy we are not adding the entry to the valid entries. We map over valid entries and display so, we will need the entries.

Reply:
i understand, but i don't see any usage of key

just wondering why we don't use Object.values here, not critial